api/CSSMatrix.json

```diff
@@ -0,0 +1,55 @@
+{
+  "api": {
+    "CSSCounterStyleRule": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSMatrix",
+        "support": {
+          "webview_android": {
+            "version_added": "45"
+          },
+          "chrome": {
+            "version_added": null
+          },
+          "chrome_android": {
+            "version_added": "45"
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": null
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
```

Comment:
I think this could be broken into two support statements, like this:

1. `version_added` set to "11" and `alternative_name` set to "WebKitCSSMatrix"
2. `version_added` set to "10" and `alternative_name` set to "MSCSSMatrix"